protocol-units/movement-rest/src/lib.rs

```diff
@@ -1,71 +0,0 @@
-use aptos_api::Context;
-
-use anyhow::Error;
-use futures::prelude::*;
-use poem::listener::TcpListener;
-use poem::{
-	get, handler,
-	middleware::Tracing,
-	web::{Data, Path},
-	EndpointExt, IntoResponse, Response, Route, Server,
-};
-use tracing::info;
-
-use std::env;
-use std::future::Future;
-use std::sync::Arc;
-
-#[handler]
-pub async fn health() -> Response {
```

Comment:
Why remove the whole service?
We rely on this endpoint for simple health checks.

We also had "malicious Richard" 😁